src/executor/Executor.cc

```diff
@@ -161,7 +170,7 @@ void Executor::setupSignalHandling() {
     sigioOccurred = 0;
     sigalrmOccurred = 0;
 
-    struct sigaction signalAction;
+    struct sigaction signalAction {};
```

Comment:
serio?
Przy intach nie piszemy pustego inicjalizatora, to po co pisać przy structach?

Reply:
```c++
#include <iostream>

struct S {
  int x;
};

int main() {
  for (size_t i = 0; i < 3; ++i) {
    S a, b{};
    std::cout << a.x << " " << b.x << std::endl;
    a.x = b.x = i;
  }

  for (size_t i = 0; i < 3; ++i) {
    int a, b{};
    std::cout << a << " " << b << std::endl;
    a = b = i;
  }
}
```
```bash
teqwve ~ $ g++ test.cc 
teqwve ~ $ ./a.out 
1347568336 0
0 0
1 0
1347567776 0
0 0
1 0
```